### ISSUE TITLE
[Work Item] CVE-2023-23397 script should recommend least privilege model

### ISSUE THREAD

**Issue by jimthemcp** [microsoft/CSS-Exchange#1589]
**Describe the work**
The documentation states that the Global Admin role is required to create the application in Azure AD. It should recommend the Application Admin role as it has fewer privileges in the tenant. They could then add a more generic user as an Application owner and use that account to run the audit and cleanup.

Exchange Org management is also excessive permissions to Exchange. The account only needs read permission to run the Get-Mailbox cmdlet. They could use View Only Org Management or create a custom role (probably too difficult for smaller orgs).

**Additional Context**
Some customers are questioning the level of permissions in the requirements.

**Comment by MeanDoubleStain** [comment]
Am Global Admin ánd Organization Management still not authorized to connect to EWS.

Unable to connect to EWS endpoint. Please make sure you have enter valid credentials. Inner Exception

The request failed. The remote server returned an error: (403) Forbidden.

**Comment by jmalco-bp** [comment]
THANK YOU for this fix. This should be added to the FAQ!

**Comment by MeanDoubleStain** [comment]
> THANK YOU for this fix. This should be added to the FAQ!

How can we make that happen?

**Comment by jmalco-bp** [comment]
@lusassl-msft would it be possible to add this fix to the FAQ?

**Comment by MeanDoubleStain** [comment]
I forked and edited in a proposal for an update to the FAQ (nevereverdidthisbefore):

## In OnPrem I get errors loging in to EWS

In Exchange 2016 onprem, with an account that's Global Admin ánd Exchange Organization Management role, that cán login to EWS, but with Powershell you get these errors:

Unable to connect to EWS endpoint. Please make sure you have enter valid credentials. Inner Exception

The request failed. The remote server returned an error: (403) Forbidden.

Try the following:

Get-Mailbox -ResultSize Unlimited | .\CVE-2023-23397.ps1 -Environment Onprem -EWSServerURL https://127.0.0.1/EWS/Exchange.asmx -IgnoreCertificateMismatch

**Comment by MeanDoubleStain** [comment]
Thanks Lukas,

Thank you for very much for looking into it, and adding it. Glad we could
attribute.

In our case we didn't have a problem with the allow list, our setup is a
very basic onprem DAG. We looked at everything and kept running in to this,
that's how we came to posting it. Eventually it was my collegue Leon Dekker
who combined our findings into this solution. I was then encouraged to post
it to the FAQ by a community member.

Groet, Rick Glorie
